### DISCUSSION TITLE
Is it possible to get links to consider aliases?

### DISCUSSION THREAD

**Discussion by waffledork** [mProjectsCode/obsidian-meta-bind-plugin#447] [Q&A] [answered]
I'm using Obsidian for some TTRPG (D&D 5e) worldbuilding and am running into an issue with Meta Bind links.

I've got an `INPUT[Race][suggester:race]` to allow me to select from a list of 5e Races and store it into the "race" frontmatter. When viewing it, I'm using `VIEW[{race}][link]` to display a link to the note about that particular race. It works great when the name of the race is the name of the file (e.g. "Dwarf", "Elf", etc.) but when it comes to variants, it doesn't recognize the note.

For example, I've got a note called "dwarf-hill" and it has an alias of "Dwarf (Hill)". When "Dwarf (Hill)" is selected via the suggester, Meta Bind displays it as a link in the View but when I hover the mouse cursor over it, it says that note doesn't exist.

Is there a setting to get Meta Bind to recognize aliases when it comes to linking?

**Comment by mProjectsCode** [comment, marked as answer]
No, there is not, and for technical reasons, this will probably never be a thing.

**Reply by waffledork** [reply]
Good to know. Thank you for answering!

I should still be able to set up an Input Field Template with value/name pairs to manually point the selections to the proper note name to get the link to work, right? Is there a way to set up the View with value/name pairs to show the chosen name despite the provided note name from the Input?

**Reply by mProjectsCode** [reply]
No, you must dip your toes into JavaScript for such a reverse mapping.

**Reply by waffledork** [reply]
Thank you!
